### ISSUE TITLE
rmetasim()

### ISSUE THREAD

**Issue by JuliaPalacios** [georgeshirreff/multiNe#12]
Also @ewersSaucedo 

Do we need rmetasim? is it a cran? should we add as dependency?

**Comment by JuliaPalacios** [comment]
Ok, I added it as dependency but I still need your file: 
"rmetasim2adegenet.R"

landscape.make.genind() is defined in that file?

**Comment by JuliaPalacios** [comment]
So.. in order to keep working and testing my changes, I commented many things in change 53443e3baea1a5b962ab55dec24f5cd51f810b1f
with label "temporary", so @ewersSaucedo let me know when you add the code so that I can undo the commenting with #?

**Comment by EwersAquaGenomics** [comment]
Sorry, guys, I was in meetings all day (and still am). First chance to check emails. I hope I can get to it tonight. We should not need rmetasim because I used it only to make the data files for the examples. Sorry to be short! Running in between meetings!

Christine

**Comment by EwersAquaGenomics** [comment]
Hi Julia,
I suppose by now you have dealt with all those changes.This script generates simulated data, but I don’t think we need this script anymore, as I submitted the data files directly. 

Sorry that I couldn’t be useful yesterday! Today is a little bit better, but still heavy on meetings.

Christine

**Comment by JuliaPalacios** [comment]
No problem. Do you think it is better if I remove that file?

**Comment by EwersAquaGenomics** [comment]
Yes, do remove it, especially if it is being difficult. I included it to show how the simulations were done but I don’t think it is a super-important file.

Christine
